aiida/cmdline/verdilib.py

```diff
@@ -805,17 +822,24 @@ def _quicksetup_cmd(self, email, first_name, last_name, institution, backend, db
 
         for process in valid_processes:
 
-            default_profile = default_profiles.get(process, '')
-            default_override = False
-
-            if default_profile:
-                default_override = click.confirm("The default profile for the '{}' process is set to '{}': "
-                                                 "do you want to set the newly created '{}' as the new default? (can be reverted later)"
-                                                 .format(process, default_profile, profile_name))
+            # if the user specifies whether to override that's fine
+            if set_default in [True, False]:
+                _set_default = set_default
+            # otherwise we may need to ask
+            else:
+                default_profile = default_profiles.get(process, '')
+                if default_profile:
+                    _set_default = click.confirm("The default profile for the '{}' process is set to '{}': "
+                                                     "do you want to set the newly created '{}' as the new default? (can be reverted later)"
+                                                     .format(process, default_profile, profile_name))
+                # if there are no other default profiles, we don't need to ask
+                else:
+                    _set_default = True
 
-            if not default_profile or default_override:
+            if _set_default:
                 set_default_profile(process, profile_name, force_rewrite=True)
 
+    #TODO (ltalirz) db-related functions should be moved to the base level of verdilib
```

Comment:
Please add an issue for this (instead of a comment) and assign it to me unless you are interested in fixing it yourself

Reply:
Done.